### ISSUE TITLE
Support hour on visibleRange option

### ISSUE THREAD

**Issue by trungk18** [fullcalendar/fullcalendar#4658]
Hi,

Thanks for writing this library. I am using the resourceTimeline view.

Currently I am trying to show the visible range of the calendar based on the hour, not the day. For instance I need to show only the timeslot from `4am` until `8pm`. You can refer to the red box on the photo.

![visible-range](https://user-images.githubusercontent.com/6767322/56888050-b0cec600-6aa5-11e9-9944-1163951dfeb2.png)

The default behavior is always display 12am to 11:59pm. I tried the `visibleRange` option but no luck since It only accept date value, based on the example.

Are there something that I can do? Sorry If it is supported but I am not able to find it property.

**Comment by acerix** [comment]
I believe your are looking for the [minTime](https://fullcalendar.io/docs/minTime) and [maxTime](https://fullcalendar.io/docs/maxTime) settings.

https://codepen.io/anon/pen/mggjRM?&editable=true&editors=001

**Comment by trungk18** [comment]
Thanks @acerix,

I tried, it worked perfectly for one day. But If I have a date range that span more than one day, it doesn't work as my expectation anymore. 

My use case also need to display for instance from 2019-03-29T07:00:00 to 2019-03-30T21:00:00. Meaning from 7h to midnight of 2019-03-29. And from 00:00 of 2019-03-30 to 21:00. The below gif is when I try to set the visibleRange span multiple days. I couldn't make it trim at the start and the end time.

![2019-04-30_14-57-37](https://user-images.githubusercontent.com/6767322/56945099-72deaa00-6b58-11e9-9c1a-e1fb43ec649f.gif)

If I set the minTime "07:00" and maxTime "21:00", it will apply for the both days. So that the duration from 2019-03-29T21:00:00 to 2019-03-30T07:00:00 will be missing on the calendar. Like the below image.

![image](https://user-images.githubusercontent.com/6767322/56944972-06fc4180-6b58-11e9-966c-96701fb9c48f.png)

Do you have any idea how to archive that? 

Thanks for your support.

**Comment by acerix** [comment]
You could set to `maxTime` to "45:00" which will make the range end to 21:00 the next day. (45:00 = 21:00 + 24 hours)

https://codepen.io/anon/pen/gyVzNB?&editable=true&editors=001

**Comment by trungk18** [comment]
Thanks @acerix ,

I didn't know that you can set the value that exceed 24:00. It appear to be working.

https://codepen.io/trungk18/pen/JVgayK?editors=0010

If I only set the minTime and maxTime, the date on top will not appear. I mean this red area. I wanted to show this date bar too.

![image](https://user-images.githubusercontent.com/6767322/57119306-a6f5dd00-6d9b-11e9-9940-76d95b6e46ff.png)

If I set the visibleRange, the date bar appear but it behaves weirdly. 

https://codepen.io/trungk18/pen/mgNGqq?editors=0010

Basically I want to show the range from 2019-03-16T05:00:00 to 2019-03-17T08:00, but as you can see on the pen above, It will show until 2019-03-18T08:00. Which is another one day. 

Are there any option to show the date bar on top when using minTime and maxTime? I couldn't find any usable option under https://fullcalendar.io/docs/date-display

Thank you.

**Comment by acerix** [comment]
You can set [slotLabelFormat](https://fullcalendar.io/docs/slotLabelFormat
) to customize those bars:

https://codepen.io/anon/pen/EzYXNp?editors=0010

**Comment by trungk18** [comment]
Thanks man, 

That's all I need for now. Appreciate your support.
